Update concepts-symmetric-key-attestation.md

LeelaRajesh-Sayana opened the pull request:
Fixing deprecated Azure CLI command
Resolves : [MicrosoftDocs/azure-docs#104404](https://github.com/MicrosoftDocs/azure-docs/issues/104404)

LeelaRajesh-Sayana committed:
Update concepts-symmetric-key-attestation.md

Fixing deprecated Azure CLI command

kgremban commented on articles/iot-dps/concepts-symmetric-key-attestation.md:
````diff
@@ -76,7 +76,7 @@ Replace the value of `--key` argument with the **Primary Key** from your enrollm
 Replace the value of `--registration-id` argument with your registration ID.
 
 ```azurecli
````
Please also update the link to the CLI reference in line 72. 

It should point to /cli/azure/iot/dps/enrollment-group#az-iot-dps-enrollment-group-compute-device-key

LeelaRajesh-Sayana replied:
Thank you, Kelly, for pointing this out. I have updated the link.

LeelaRajesh-Sayana committed:
Update concepts-symmetric-key-attestation.md

Updated the link to the CLI reference in line 72 to point to /cli/azure/iot/dps/enrollment-group#az-iot-dps-enrollment-group-compute-device-key
